cider/tests/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
+include_directories(${CMAKE_SOURCE_DIR}/thirdparty/velox)
```

Comment:
please consider combine with cider/function/CMakeLists.txt to move the needed include path to top cmake file

Reply:
Done.